DISCUSSION.md

```diff
@@ -14,7 +14,7 @@ this branch:
 - [ ] Finish re-incorporating mobile sidebar
 - [ ] Re-integrate google-analytics
 - [ ] Re-incorporate `redirects.json`
-- [ ] Populate page title in `server.jsx` (fernando)
+- [x] Populate page title in `server.jsx` (fernando)
```

Comment:
Drop this to resolve the conflict (see comments on other other PRs for context).